hack/e2e.sh

```diff
@@ -1,52 +1,49 @@
+TEST_JUNIT_REPORT_FILE="${TEST_JUNIT_REPORT_FILE:-tests-junit.xml}"
+TEST_JUNIT_REPORT_FILE_PATH="${TEST_JUNIT_REPORT_FILE_PATH:-$TEST_OUTPUT_DIR/$TEST_JUNIT_REPORT_FILE}"
 
-export DEPLOY_SCRIPT="${DEPLOY_SCRIPT:-deploy-e2e.sh}"
-export TEST_SCRIPT="$ROOT_DIR/hack/run-e2e-tests.sh"
+mkdir -p "$TEST_OUTPUT_DIR"
 
-export TEST_LOG_FILE="${TEST_LOG_FILE:-e2e-tests.log}"
-export DEPLOY_LOG_FILE="${DEPLOY_LOG_FILE:-e2e-deploy.log}"
-export TEST_TAP_FILE="${TEST_TAP_FILE:-e2e-tests.tap}"
+export METERING_TESTING_SUITE_BASE_DIR="${METERING_TESTING_SUITE_BASE_DIR:=${ROOT_DIR}/test/e2e}"
+export METERING_NAMESPACE="${METERING_E2E_NAMESPACE:=${METERING_NAMESPACE}-e2e}"
+export METERING_SHORT_TESTS=${METERING_SHORT_TESTS:=false}
+export METERING_DEPLOY_MANIFESTS_PATH=${METERING_DEPLOY_MANIFESTS_PATH:=${ROOT_DIR}/manifests/deploy}
+export METERING_CLEANUP_SCRIPT_PATH=${METERING_CLEANUP_SCRIPT_PATH:=${ROOT_DIR}/hack/run-test-cleanup.sh}
+export TEST_OUTPUT_PATH=${TEST_OUTPUT_PATH:="$(mktemp -d)/$METERING_E2E_NAMESPACE"}
+export TEST_LOG_LEVEL=${TEST_LOG_LEVEL:="debug"}
 
 echo "\$KUBECONFIG=$KUBECONFIG"
 echo "\$METERING_NAMESPACE=$METERING_NAMESPACE"
 echo "\$METERING_OPERATOR_IMAGE_REPO=$METERING_OPERATOR_IMAGE_REPO"
 echo "\$REPORTING_OPERATOR_IMAGE_REPO=$REPORTING_OPERATOR_IMAGE_REPO"
 echo "\$METERING_OPERATOR_IMAGE_TAG=$METERING_OPERATOR_IMAGE_TAG"
 echo "\$REPORTING_OPERATOR_IMAGE_TAG=$REPORTING_OPERATOR_IMAGE_TAG"
+echo "\$TEST_OUTPUT_PATH=$TEST_OUTPUT_PATH"
 
-export DISABLE_PROMETHEUS_METRICS_IMPORTER=false
-if [ "$DEPLOY_REPORTING_OPERATOR_LOCAL" == "true" ]; then
-    export REPORTING_OPERATOR_PROMETHEUS_DATASOURCE_MAX_IMPORT_BACKFILL_DURATION='15m'
-    export REPORTING_OPERATOR_PROMETHEUS_METRICS_IMPORTER_INTERVAL="30s"
-    export REPORTING_OPERATOR_PROMETHEUS_METRICS_IMPORTER_CHUNK_SIZE="5m"
-    export REPORTING_OPERATOR_PROMETHEUS_METRICS_IMPORTER_INTERVAL="5m"
-    export REPORTING_OPERATOR_PROMETHEUS_METRICS_IMPORTER_STEP_SIZE="60s"
-
-    export REPORTING_OPERATOR_API_LISTEN="127.0.0.1:8100"
-    export REPORTING_OPERATOR_METRICS_LISTEN="127.0.0.1:8101"
-    export REPORTING_OPERATOR_PPROF_LISTEN="127.0.0.1:8102"
-
-    export METERING_PRESTO_PORT_FORWARD_PORT="8103"
-    export METERING_HIVE_PORT_FORWARD_PORT="8104"
-    export METERING_PROMETHEUS_PORT_FORWARD_PORT="8105"
-
-    export METERING_HTTPS_API="false"
-    export METERING_REPORTING_API_URL="http://$REPORTING_OPERATOR_API_LISTEN"
-    export METERING_USE_KUBE_PROXY_FOR_REPORTING_API="false"
-    export METERING_USE_ROUTE_FOR_REPORTING_API="false"
-fi
+set +e
+set +o pipefail
+echo "Running E2E Tests"
+"${ROOT_DIR}/hack/$TEST_HACK_SCRIPT" 2>&1 | tee "$TEST_LOG_FILE_PATH"
 
-if [ "$DEPLOY_METERING_OPERATOR_LOCAL" == "true" ]; then
-    export METERING_OPERATOR_CONTAINER_NAME="metering-operator-e2e"
+# attempt to create a go junit report if the package is already installed
+if [[ -f "$GOPATH"/bin/go-junit-report ]]; then
```

Comment:
Can you use the logic from `e2e-test-runner.sh` instead of this new logic for detecting go-junit-report? Previously we just looked for `go-junit-report` was in your PATH using `command -v go-junit-report`